js/customize-snapshots.js

```diff
@@ -889,6 +891,176 @@
+		icon: wp.template( 'snapshot-conflict-button' ),
+		refreshBuffer: 250,
+		controls: {},
+		pendingRequest: {},
+		notificationCode: 'snapshot_conflict',
+		notificationTemplate: wp.template( 'snapshot-notification-template' ),
+		conflictValueTemplate: wp.template( 'snapshot-conflict-value' )
+	};
+
+	/**
+	 * Handles snapshot conflict events.
+	 *
+	 * @return {void}
+	 */
+	component.handleEventForConflicts = function() {
+		api.state( 'saved' ).bind( function( saved ) {
+			if ( saved ) {
+				_.each( component.conflict.controls, function( control ) {
+					control.remove();
+				} );
+				if ( component.conflict._currentRequest ) {
+					component.conflict._currentRequest.abort();
+					component.conflict._currentRequest = null;
+				}
+				component.conflict.controls = {};
+				component.conflict.pendingRequest = {};
+
+				api.control.each( function( control ) {
+					_.each( control.settings, function( setting ) {
+						setting.unbind( component.onConflictFirstChange );
+					} );
+					control.notifications.remove( component.conflict.notificationCode );
+					component.addConflictButton( control );
+				} );
+			}
+		} );
+	};
+
+	/**
+	 * Add conflict button icon on first change and trigger handleConflictRequest.
+	 *
+	 * @param {object} control where conflict button will be added.
+	 *
+	 * @return {void}
+	 */
+	component.addConflictButton = function addConflictButton( control ) {
+		control.deferred.embedded.done( function() {
+			var onFirstChange, hasDirty, updateCurrentValue, bindFirstChange = false;
+
+			if ( ! control.setting ) {
+				return;
+			}
+
+			onFirstChange = component.onConflictFirstChange = function() {
+				_.each( control.settings, function( setting ) {
+					setting.unbind( onFirstChange );
+					component.handleConflictRequest( setting );
+				} );
+			};
+
+			updateCurrentValue = function() {
+				_.each( control.settings, function( setting ) {
+					component.updateConflictValueMarkup( setting.id, control.container );
+				} );
+			};
+
+			hasDirty = _.find( control.settings, function( setting ) {
+				return setting._dirty;
+			} );
+			if ( hasDirty ) {
+				onFirstChange();
+			} else {
+				bindFirstChange = true;
+			}
+			_.each( control.settings, function( setting ) {
+				if ( bindFirstChange ) {
+					setting.bind( onFirstChange );
+				}
+				setting.bind( updateCurrentValue );
+			} );
+		} );
+	};
+
+	/**
+	 * Handles the snapshot conflict request
+	 *
+	 * @param {object} setting to check conflicts
+	 *
+	 * @return {void}
+	 */
+	component.handleConflictRequest = function conflictRequest( setting ) {
+		if ( component.conflict._currentRequest ) {
+			component.conflict._currentRequest.abort();
+			component.conflict._currentRequest = null;
+		}
+		if ( component.conflict._debouncedTimeoutId ) {
+			clearTimeout( component.conflict._debouncedTimeoutId );
+			component.conflict._debouncedTimeoutId = null;
+		}
+		component.conflict.pendingRequest[setting.id] = setting.findControls();
```

Comment:
Okay, that was because I was running this on WP 4.5.x. 
We need to safely fall when the version is below 4.6.